File: Python/Code/PetShop.py
class PetShop:
    __ID = ""
    __nama_produk = ""
    __harga_produk = 0
    __stok_produk = 0

    def __init__(self, ID, nama_produk, harga_produk, stok_produk):
        self.__ID = ID
        self.__nama_produk = nama_produk
        self.__harga_produk = harga_produk
        self.__stok_produk = stok_produk

    def getID(self):
        return self.__ID
    
    def getnamaProduk(self):
        return self.__nama_produk
    
    def getstok(self):
        return self.__stok_produk
    
    def getharga(self):
        return self.__harga_produk

File: Java/Code/PetShop.java
public class PetShop {
    private String ID;
    private String nama_produk;
    private int harga_produk;
    private int stok_produk;

    public PetShop(){
    }
    
    public PetShop(String ID, String nama_produk, int harga_produk, int stok_produk){
        this.ID = ID;
        this.nama_produk = nama_produk;
        this.harga_produk = harga_produk;
        this.stok_produk = stok_produk;
    }

    public String getID() {
        return this.ID;
    }

    public String getnamaProduk() {
        return this.nama_produk;
    }

    public int getharga() {
        return this.harga_produk;
    }

    public int getstok() {
        return this.stok_produk;
    }
}